app/js/lib/helpers.js

```diff
@@ -0,0 +1,45 @@
+export class IconHelper {
```

Comment:
At first I thought about doing it that way, and having each of these helpers be in a separate file. But then I decided to put them together, and used `lib` since I might put something else in here, and `helpers/helpers.js` seems a little weird. In any case, it's six and half dozen.